seedutil/README.md

```diff
@@ -0,0 +1,8 @@
+# seedutil
+This utility is used to derive Ethereum keystore file from [aezeed](https://github.com/lightningnetwork/lnd/tree/master/aezeed) generated mnemonic seed.
+
+## Build
+`npm run compile:seedutil`
+
+## Usage
+[Tests](/test/jest/SeedUtil.spec.ts)
```

Comment:
Does it make sense to add the command for running this directly from the command line here?

Also for tests, I'm wondering if we want a `test:seedutil` npm script that's separate from the other jest scripts, which we can then put here. I figure eventually as we continue migrating to jest, we'll want to keep our `test:int`, `test:unit`, etc... breakdown and `test:seedutil` would fit in there.

Reply:
I didn't want to add a command line usage here because of privacy concerns (the history is recorded by default). It does make sense to add it for development purposes. I'll add it with a warning.

As for tests, yes. Good idea - now that we have more than 1 jest test suite it makes sense to start dividing them into categories.